src/App.js

```diff
@@ -1,75 +1,154 @@
 import React from 'react';
 
-function App() {
-  return (
-    <section className="todoapp">
-      <header className="header">
-        <h1>todos</h1>
-
-        <input
-          className="new-todo"
-          placeholder="What needs to be done?"
-        />
-      </header>
-
-      <section className="main" style={{ display: 'block' }}>
-        <input type="checkbox" id="toggle-all" className="toggle-all" />
-        <label htmlFor="toggle-all">Mark all as complete</label>
-
-        <ul className="todo-list">
-          <li className="">
-            <div className="view">
-              <input type="checkbox" className="toggle" id="todo-1" />
-              <label htmlFor="todo-1">sdfsdfsdf</label>
-              <button type="button" className="destroy" />
-            </div>
-          </li>
-
-          <li className="">
-            <div className="view">
-              <input type="checkbox" className="toggle" id="todo-2" />
-              <label htmlFor="todo-2">sakgjdfgkhjasgdhjfhs</label>
-              <button type="button" className="destroy" />
-            </div>
-          </li>
-
-          <li className="">
-            <div className="view">
-              <input type="checkbox" className="toggle" id="todo-3" />
-              <label htmlFor="todo-3">sddfgdfgdf</label>
-              <button type="button" className="destroy" />
-            </div>
-          </li>
-        </ul>
-      </section>
+import TodoList from './components/TodoList/TodoList';
+import Form from './components/Form/Form';
+import Filters from './components/Filters/Filters';
+
+class App extends React.Component {
+  state = {
+    todos: [],
+    completedAll: 0, // eslint-disable-line
+    active: 1,
+  }
+
+  AddTodo = (inputFormValue) => {
```

Comment:
```suggestion
  addTodo = (value) => {
```
нижний регистр
зачем усложнять название переменной?

я ниже вижу что мы добавляем объект, тогда это 
```suggestion
  addTodo = (todo) => {
```